waku/waku_keystore/keystore.nim

```diff
@@ -31,7 +31,12 @@ proc createAppKeystore*(
     f.write(separator)
     ok()
   except CatchableError:
-    err(AppKeystoreError(kind: KeystoreOsError, msg: getCurrentExceptionMsg()))
+    err(
+      AppKeystoreError(
+        kind: KeystoreOsError,
+        msg: "error in createAppKeystore: " & getCurrentExceptionMsg(),
```

Comment:
If it will be for the public ... like logs for our node-runners I would rather formalize like:
`error while writing keystore:`